pkg/cmd/server/admin/create_kubeconfig.go

```diff
@@ -183,6 +186,7 @@ func (o CreateKubeConfigOptions) CreateKubeConfig() (*clientcmdapi.Config, error
 		CurrentContext: o.ContextNick,
 	}
 
+	fmt.Fprintf(o.GetOut(), "Generating '%s' API client config as %s\n", o.UserNick, o.KubeConfigFile)
```

Comment:
Add an indication of whether we blew away an existing file.  Two different sentences?